app/assets/stylesheets/find_application.scss

```diff
@@ -13,6 +13,7 @@
 @import "components/shared/accordion";
 
 @import "components/find/accordion";
+@import "components/find/call-out-box";
```

Comment:
Can this be just one word, `callout`, to align with the majority of UI kit naming schemes please?

https://www.google.com/search?q=call+out+component&oq=call+out+component&gs_lcrp=EgZjaHJvbWUyCQgAEEUYORiABDIICAEQABgWGB4yCAgCEAAYFhgeMggIAxAAGBYYHjINCAQQABiGAxiABBiKBTIKCAUQABiABBiiBDIGCAYQRRg8MgYIBxBFGDzSAQgzODk1ajBqN6gCALACAA&sourceid=chrome&ie=UTF-8